tools/travis-scripts/generate-jsbindings.sh

```
@@ -49,16 +49,15 @@ if [ "$GEN_JSB"x != "YES"x ]; then
     exit 0
 fi
 
-#Set git user
-git config --global user.email ${GH_EMAIL}
-git config --global user.name ${GH_USER}
-
 # Update submodule of auto-gen JSBinding repo.
 pushd "$GENERATED_WORKTREE"
 
 git checkout -B master
+#Set git user
+git config user.email ${GH_EMAIL}
+git config user.name ${GH_USER}
 #Set remotes
-git remote add upstream https://${GH_USER}:${GH_PASSWORD}@github.com/folecr/cocos2dx-autogen-bindings.git 2> /dev/null > /dev/null
+git remote add upstream https://${GH_USER}:${GH_PASSWORD}@github.com/folecr/cocos2dx-autogen-bindings.git > /dev/null
```

Oh, I didn't realize why it was redirected.

I had the problem that this command was failing, which was resulting in a silent failure (for no apparent reason).  

I just did some investigation and it looks like 'git remote add' doesn't output anythign to the stderr or stdout in normal circumstances.  

In any case my problem was fixed by check for GH_USER and GH_PASSWORD prior to even calling this script.